File: src/j1bwmfinddifference/Go/main.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	// Example input strings
	string1List := []string{"wxyz", "cbda", "jlkmn", "courae", "hello"}
	string2List := []string{"zwxgy", "abc", "klmn", "couearg", "helo"}

	for i := 0; i < len(string1List); i++ {
		fmt.Printf("%d.\tString 1 = %s\n", i+1, string1List[i])
		fmt.Printf("\tString 2 = %s\n", string2List[i])
		fmt.Printf("\n\tThe extra character is at index: %d\n", extraCharacterIndex(string1List[i], string2List[i]))
		fmt.Printf("%s\n", strings.Repeat("-", 100))
	}
}

File: src/j1bwmfinddifference/JS/main.js
// File: main.js

const extraCharacterIndex = require('./differenceFinder.js');

const string1List = ["wxyz", "cbda", "jlkmn", "courae", "hello"];
const string2List = ["zwxgy", "abc", "klmn", "couearg", "helo"];

for (let i = 0; i < string1List.length; i++) {
  console.log(`${i + 1}.\tString 1 = ${string1List[i]}`);
  console.log(`\tString 2 = ${string2List[i]}`);
  console.log(`\n\tThe extra character is at index: ${extraCharacterIndex(string1List[i], string2List[i])}`);
  console.log('-'.repeat(100));
}